default/data/ui/views/overview.xml

```diff
@@ -124,7 +124,7 @@
   <row>
 
     <panel>
-      <html><p><i><b>Note:</b> The data presented in the below panels may lag by 30 minutes.</i></p></html>
+      <html><p><i><b>Note:</b> The data presented in the below panels may lag by 6 hours.</i></p></html>
```

Comment:
Please update to say `The data in the panels below are updated every 6 hours.  For the most recent information, visit the Trends page of the GreyNoise Visualizer`

Reply:
Resolved.